packages/opc-styles/variables/_1-definitions.scss

```diff
@@ -4,3 +4,19 @@ $prefix: opc !default;
 
 /// adding support for css custom property by default
 $USE-FALLBACK: true;
+
+/// Global variables for Gap/ Spacing in layout
+/// Small Space
+$opc-global-spacer-sm-4px: 4px;
```

Comment:
we practically can't cover variables for every margin/padding.
Can we just provide something simple and smaller like
--opc-global--spacer--sm:0.5rem;
--opc-global--spacer--md:1rem;
--opc-global--spacer--lg:1.5rem;
--opc-global--spacer--xl:2rem;
--opc-global--spacer--2xl:3rem;
--opc-global--spacer--3xl:4rem;

Reply:
@gisumit if user wants something extra they can directly use the component's local variable.
For eg: a local component variable for footer can be `--op-c--Margin: var(--op-global--spacer--md);` (this will be the default)
if user wants something like 0.25 then they can do `--op-c--Margin:0.25` directly from the component

Reply:
@sayak-sarkar @deshmukhmayur let me know if you like this idea. thats how other libraries are also managing their spacer variables. the vaariables should be standardized so that devs dont have to remember long variable names. for ex: we can standardize that footer component uses op--global--spacer--xs margin